tutorial/templates/problem-template.mt

```diff
@@ -7,10 +7,11 @@
 	<title><%= $filename %></title>
 
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet">
-	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/codemirror@5.65.11/lib/codemirror.min.css">
-	<script src="https://cdn.jsdelivr.net/npm/codemirror@5.65.11/addon/runmode/runmode-standalone.min.js" defer>
+	<script
+		src="https://cdn.jsdelivr.net/npm/@openwebwork/pg-codemirror-editor@0.0.1-beta.26/dist/pg-codemirror-editor.js"
+		defer
+	>
```

Comment:
I assume there is no better way than to hard code the version here, just feel this could get overlooked during a release.

Reply:
Unfortunately, there isn't.  This isn't like the packages installed by npm where the version is automatically handled by the package system.  I don't really like it either, although, it was previously a hard coded version of the CodeMirror 5 package that was there.  So it doesn't really change anything with that.

I plan to remove the beta version before the next release, and once there is a stable version here it should be fine.